ci/test.sh

```diff
@@ -145,6 +145,32 @@ case $provider in
     ;;
 esac
 
+# test internal communication
+case $provider in
+  aws)
+  convox apps delete ci1 --wait
+  convox rack params set Internal=Yes
+
+  cd $root/examples/httpd
+  convox apps create ci1 --wait
+  convox apps | grep ci1
+  convox apps info ci1 | grep running
+  convox deploy -a ci1 --wait
+  convox apps info ci1 | grep running
```

Comment:
do you need to delete the app and create it again after setting the Internal as Yes? Can you just set the param and deploy the app again?

```suggestion
  convox rack params set Internal=Yes

  cd $root/examples/httpd
  convox deploy -a ci1 --wait
  convox apps info ci1 | grep running
```

Reply:
I couldn't but I'll try again

Reply:
yeah, @Twsouza I don't know why but the app could not resolve the host.

Reply:
yeah, I did use the `--wait`. But only works if I recreate the app

Reply:
@yuriadams can you add this to our documentation?
I appreciate it.